src/app/(auth)/components/LoginBgImage.tsx

```diff
@@ -0,0 +1,13 @@
+
+export default function LoginBgImage() {
+    return (
+      <div
+        className="hidden bg-cover lg:block lg:w-3/5 h-screen"
+        style={{
+          backgroundImage: `url("https://cdn.leonardo.ai/users/58b4ebe8-31b7-423d-8587-b03acce30173/generations/5d8ef9a3-f1fa-4373-832b-76038240b7b8/DreamShaper_v7_a_heros_journey_1.jpg")`,
```

Comment:
If possible we can just download this image and add it to our project (for future)